Toolbar button tooltips for keyboard users

mmotyczynska opened the pull request:
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Fix (toolbar): Adds toolbar button tooltips for keyboard users. Closes #5581.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

mlewand commented on packages/ckeditor5-theme-lark/theme/ckeditor5-ui/components/toolbar/toolbar.css:
```diff
@@ -99,6 +100,14 @@
 		border: 0;
 	}
+
+	& .ck.ck-button {
```
If we move it I would consider moving the unit tests to `buttonview.js` file (if possible).

I have a feeling that we'll be also able to avoid creating the editor there to test this behavior.

mlewand reviewed (changes requested):
I reverted the source code change with `git co origin/master packages/ckeditor5-theme-lark/theme/ckeditor5-ui/components/toolbar/toolbar.css` command and after running `yarn run test -f ui` only one of two unit tests is failing.

The second should fail too as it's for a regular button and should fail too:

```
SUMMARY:
√ 1003 tests completed
× 1 test failed

FAILED TESTS:
  Tooltip in toolbar button
    × which is a dropdown, displays the tooltip when the button is focused
      Chrome 102.0.5005.63 (Windows 10)
    AssertionError: expected false to be true

      + expected - actual

      -false
      +true

    at Context.eval (./packages/ckeditor5-ui/tests/toolbar/toolbarview.js:1347:118)


Error: Karma finished with "1" code.
```

Interestingly I can see that the tooltip is **not added** to a regular button when testing this manually (given that your change is reverted) so it looks like the test is off.

mmotyczynska commented:
I moved the code to `button.css` and `buttonview.js` respectively as you suggested. I also changed how the unit test works, I'm not creating an editor anymore and instead of `isVisible` I'm using `getComputedStyle` to check the tooltip's visibility.

mlewand reviewed (commented):
There are some minor things to be polished, other than that looks good. I'll handle these changes since I have the branch already open.

mlewand reviewed (approved):
LGTM. I'll wait with the merge for CI.

wimleers commented:
Yay, this fixed one of the aspects of https://www.drupal.org/project/drupal/issues/3283800 🥳 🚀
